tests/CrawlerRunner/ip_address.py

```diff
@@ -10,6 +12,16 @@
 from tests.mockserver import MockServer, MockDNSServer
 
 
+# https://stackoverflow.com/a/32784190
+def createResolver(servers=None, resolvconf=None, hosts=None):
+    if hosts is None:
+        hosts = b'/etc/hosts' if platform.getType() == 'posix' else r'c:\windows\hosts'
```

Comment:
*Ideally* this should detect the Windows dir path at run time

Reply:
Indeed. However, what we really care for here is `servers` getting picked up. The code for `hosts` is just so that we have a hosts value to pass to `hostsModule.Resolver`.